tests/integ/scala/test_permanent_udf_suite.py

```diff
@@ -19,13 +19,6 @@
 from snowflake.snowpark.functions import call_udf, col
 from tests.utils import TempObjectType, TestFiles, Utils
 
-pytestmark = [
-    pytest.mark.udf,
-    pytest.mark.skipif(
-        condition="config.getvalue('local_testing_mode')",
-    ),
-]
-
```

Comment:
we shall keep the mark `pytest.mark.udf`?

Reply:
Oh good catch, fixing